kobo/apps/subsequences/tests/test_models.py

```diff
@@ -146,6 +146,8 @@ def test_retrieve_data_with_invalid_arguments(self):
                 self.asset, submission_root_uuid=None, prefetched_supplement=None
             )
 
+    # skip until we actually fill out or delete these tests
```

Comment:
Does this comment belong to this unit test only or the two below marked as skipped?

Reply:
Applies to both. I can clarify that.